Source/WebCore/page/LocalFrame.cpp

```diff
@@ -1311,6 +1311,20 @@ void LocalFrame::didAccessWindowProxyPropertyViaOpener(WindowProxyProperty prope
 
 #endif
 
+String LocalFrame::customUserAgent() const
+{
+    if (RefPtr documentLoader = loader().activeDocumentLoader())
+        return documentLoader->customUserAgent();
+    return nullString();
```

Comment:
This is really only needed when you return a const String&.  Here, you return a String, so you can just return { }.  Ditto below.